examples/2d/texture_atlas.rs

```diff
@@ -207,8 +207,8 @@ fn create_texture_atlas(
     textures: &mut ResMut<Assets<Image>>,
 ) -> (TextureAtlasLayout, Handle<Image>) {
     // Build a texture atlas using the individual sprites
-    let mut texture_atlas_builder =
-        TextureAtlasBuilder::default().padding(padding.unwrap_or_default());
+    let mut texture_atlas_builder = TextureAtlasBuilder::default();
+    texture_atlas_builder.padding(padding.unwrap_or_default());
```

Comment:
I think we can remove this assignment now and chain it together?

Reply:
Mm, gotcha.

This does feel like a step backwards in the API. Would be a bit of an unfortunate migration for users to need to split up chained calls into separate statements.